src/control_helpers/air_actions_tracking.rs

```diff
@@ -134,6 +134,48 @@ impl TnuaSimpleAirActionsCounter {
         }
     }
 
+    /// Resets the air actions counter to a specific count, excluding the current action.
+    ///
+    /// This method allows you to manually set the count of air actions (excluding the current
+    /// action) to a specified value. Use this when you need to synchronize or initialize the air
+    /// actions count to a specific state.
+    ///
+    /// # Arguments
+    ///
+    /// * `count` - The new count to set for air actions, excluding the current action.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use bevy_tnua::control_helpers::TnuaSimpleAirActionsCounter;
+    /// let mut air_actions_counter = TnuaSimpleAirActionsCounter::default();
+    ///
+    /// // Reset the air actions count to 3 (excluding the current action). should also be updated as stated in TnuaAirActionsTracker
+    /// air_actions_counter.reset_count_to(3);
+    /// ```
+    pub fn reset_count_to(&mut self, count: usize) {
+        self.actions_including_current = count;
+    }
+
+    /// Resets the air actions counter to zero, excluding the current action.
+    ///
+    /// This method is a convenience function equivalent to calling `reset_count_to(0)`. It sets
+    /// the count of air actions (excluding the current action) to zero.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use bevy_tnua::control_helpers::TnuaSimpleAirActionsCounter;
+    /// let mut air_actions_counter = TnuaSimpleAirActionsCounter::default();
+    ///
+    /// // Reset the air actions count to zero (excluding the current action). should also be updated as stated in TnuaAirActionsTracker
+    /// air_actions_counter.reset_count();
+    /// ```
+    pub fn reset_count(&mut self) {
+        self.reset_count_to(0);
```

Comment:
Is zero really the correct number?

When I see this feature, I think about these game that allow you to double-jump and have some pickups you can collect in the air that allow you to jump again, after your first air jump, without touching the ground.

Say we do this here, with `TnuaSimpleAirActionsCounter`. You jump (or free-fall), and the counter increases from 0 (landed) to 1. You jump again in the air - it goes to 2. You're out of air jumps, but then you collect the powerup and `controller.reset_count()` is called. The counter goes to 0. So you jump again - and the counter goes to 1. But since it's 1 - you get an extra air jump without collecting the token. I doubt this is what you'd usually want...

I think one would usually want to reset it to 1, not 0. I'm not sure if we really need an helper method just for passing a simple constant non-magic number, but since the intuition is to reset to 0, maybe having a method that resets to 1 will make it harder to make this mistake.

Also, if we reset it to 1, we need to make sure that `reset_count_to` only changes the counter if it's non-zero. If the character is on the ground and we "reset" it to 1, it'll stay 1 because we'll not have a `TnuaAirActionsUpdate::JustLanded` until the character leaves the ground and lands again. Imagine picking up a low hanging extra-jump token, and what it actually does is take a jump away from you?